proxy/cc-proxy.spec-template

```diff
@@ -67,20 +68,20 @@ tar -C local -xzf ../SOURCES/go%{GO_VERSION}.linux-amd64.tar.gz
 export GOROOT=$HOME/rpmbuild/BUILD/local/go
 export PATH=$PATH:$HOME/rpmbuild/BUILD/local/go/bin
 export GOPATH=$HOME/rpmbuild/BUILD/go/
-export COMMIT=%{version}
+#export COMMIT=%{version}
```

Comment:
Better to remove instead of comment lines.

Reply:
oops